components/api_calls/build_calls.js

```diff
@@ -6,14 +6,18 @@ export async function getBuilds(searchParams) {
     for (const key in searchParams) {
         let value = searchParams[key];
 
-        if ((value[0] === "\"" && value[value.length - 1] === "\"") || (value[0] === "'" && value[value.length - 1] === "'")) {
-            value = value.substring(1, value.length - 1);
+        // Check if the value is enclosed in double or single quotes.
+        const isEnclosedInQuotes = (value[0] === "\"" && value[value.length - 1] === "\"") || (value[0] === "'" && value[value.length - 1] === "'");
```

Comment:
Hmm couldn't a regex be an easier way to check?

Reply:
right! my bad